tests/integration/defs/triton_server/test_triton.py

```
@@ -508,7 +508,16 @@ def test_cpp_unit_tests(tritonserver_test_root, test_name, llm_root):
 
     run_shell_command(
         f"cd {llm_root}/triton_backend/inflight_batcher_llm/build && "
-        f"cmake .. -DTRTLLM_DIR={llm_root} -DCMAKE_INSTALL_PREFIX=install/ -DBUILD_TESTS=ON  -DUSE_CXX11_ABI=ON -DTRITON_COMMON_REPO_TAG=r25.05 -DTRITON_CORE_REPO_TAG=r25.05 -DTRITON_THIRD_PARTY_REPO_TAG=r25.05 -DTRITON_BACKEND_REPO_TAG=r25.05 "
+        # Get the value of TRITON_BASE_TAG from docker/Dockerfile.multi
+        triton_short_tag = main
```

Ruff (0.12.2)

SyntaxError: Expected ',', found name

---

SyntaxError: Expected ')', found newline

---

SyntaxError: Unexpected indentation

---

Line too long (293 > 120)

(E501)